src/js/index.js

```diff
@@ -104,3 +104,22 @@ window.addEventListener('load', () => {
 
 	cascade.call(window, true);
 });
+
+var mode = document.getElementsByClassName("fas");
```

Comment:
We use let/const instead of var. You want to use const in this case.

Also, as seen upper in the code, we grouppes all query selected elements together to keep the code dry.